server/go/server.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+  "encoding/json"
+  "fmt"
+  "io/ioutil"
+  "net/http"
+  "os"
+
+  "github.com/foolin/goview/supports/echoview"
+  "github.com/joho/godotenv"
+  "github.com/labstack/echo"
+  "github.com/labstack/echo/middleware"
+  "github.com/stripe/stripe-go"
+  "github.com/stripe/stripe-go/customer"
+  "github.com/stripe/stripe-go/sub"
+  "github.com/stripe/stripe-go/webhook"
+)
+
+// CreateCustomerData represents data passed in request to create customer
+// object.
+type CreateCustomerData struct {
+  PaymentMethodID string `json:"payment_method"`
+  Email           string `json:"email"`
+}
+
+// GetSubscriptionData represents data passed in request to retrieve
+// subscription.
+type GetSubscriptionData struct {
+  SubscriptionID string `json:"subscriptionId"`
+}
+
+// PublicKey returned to client.
+type PublicKey struct {
+  PublicKey string `json:"publicKey"`
+}
+
+func main() {
+  err := godotenv.Load("../../.env")
+
+  if err != nil {
+    fmt.Println("Error loading .env file")
+  }
+
+  fmt.Println(os.Getenv("STRIPE_SECRET_KEY"))
+
+  stripe.Key = os.Getenv("STRIPE_SECRET_KEY")
+  e := echo.New()
+  e.Use(middleware.Logger())
+  e.Use(middleware.Recover())
+  e.Renderer = echoview.Default()
+  e.Static("/", "../../client")
```

Comment:
Can we use STATIC_DIR set in the env? So that downloading with the CLI won't break it

Reply:
good call. updated